librosa/core/notation.py

```diff
@@ -122,16 +123,17 @@
 
 
 # Pre-compiled regular expressions for note and key parsing
+KEY_RE = re.compile(
+    r"^(?P<tonic>[A-Ga-g])" r"(?P<accidental>([#♯b!♭𝄪𝄫]*))" r":(?P<scale>(maj|min)(or)?)$"
+)
```

Comment:
I think you're right, since we have ventured into the forbidden world of key signatures like `C#b##:min`; might as well also allow natural symbols.
```suggestion
KEY_RE = re.compile(
    r"^(?P<tonic>[A-Ga-g])" r"(?P<accidental>([#♯b!♭𝄪𝄫♮]*))" r":(?P<scale>(maj|min)(or)?)$"
)
```